rtgui/lensgeom.cc

```diff
@@ -113,3 +113,19 @@ void LensGeometry::setBatchMode (bool batchMode)
     ToolPanel::setBatchMode (batchMode);
     removeIfThere (this, autoCrop);
 }
+
+void LensGeometry::disableAutoFillIfActive ()
+{
+
+    if (!batchMode) {
+        if (fill->get_active()) {
+            fillConn.block (true);
+            fill->set_active(false);
+            if (listener) {
+                listener->panelChanged (EvTransAutoFill, M("GENERAL_DISABLED"));
```

Comment:
This line causes the release build to hang when opening a file, and the debug build prints:
`MyMutex already locked!`
If I comment out this line, no problem. But we need feedback via history so the user knows what's happening.